### PR TITLE
V.26.1

### PR DESCRIPTION
**Pull request by ericgaspar**
- *Update to upstream version*
- *Add README_fr*
- *Cleaning up*

## Package_check results

[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/wemawema_ynh%20PR3%20(ericgaspar)/)

### CODE REVIEW

**Review by kay0u — APPROVED** [review]
I took a very quick look, LGTM.

File: LICENSE [hunk 1]
```diff
@@ -1,21 +1,674 @@
-The MIT License (MIT)
-
-Copyright (c) 2017 Luc Didry
```
Comment (alexAubin) [inline LICENSE]: Well legally speaking I don't know if we're allowed to just yolochange the license just like that but idk

**Review by alexAubin — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (ericgaspar) [reply LICENSE]: The upstream is GNU Affero General Public License v3.0.
Let's put it back like it was (MIT).

Comment (ldidry) [reply LICENSE]: The upstream software is AGPLv3, but I choose to put the scripts of the package under MIT License.
As I’m the author of the upstream software and the original author of the YNH packaging, I think I can choose any license I want, so let’s keep it that way (AGPLv3 + MIT) :grin:

Comment (ericgaspar) [reply LICENSE]: You are absolutely right. It is as it was before.

**Pull request merged**